### PR TITLE
refactor(planner): refactor css as per code guidelines

### PR DESCRIPTION
**Pull request by SMahil**
refactor css as per code guidelines http://codeguide.co/#css
Closes: #1880

### CODE REVIEW

**Comment by fabric8cd** [comment]
@SMahil snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1892-fabric8-planner.int.rdu2c.fabric8.io

File: src/assets/stylesheets/_dragula.scss [hunk 1]
```diff
@@ -1,35 +1,32 @@
...
+  margin: 0 !important;
+  background-color: $color-pf-white;
   cursor: -webkit-grabbing;
   cursor:    -moz-grabbing;
   cursor:         grabbing;
-  margin: 0!important;
-  z-index: 9999!important;
-  opacity: 1;
-  background-color: $color-pf-white;
+  opacity:    1;
```
Comment (mindreeper2420) [inline src/assets/stylesheets/_dragula.scss]: remove extra spacing

File: src/assets/stylesheets/_dragula.scss [hunk 2]
```diff
@@ -1,35 +1,32 @@
...
   -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=80)";
-  filter: alpha(opacity=80);
-}
-.gu-hide {
-  display: none!important;
+  filter:     alpha(opacity=80);
```
Comment (mindreeper2420) [inline src/assets/stylesheets/_dragula.scss]: add space between `p` and `{`

Comment (SMahil) [inline src/assets/stylesheets/_dragula.scss]: which p?

**Comment by fabric8cd** [comment]
@SMahil snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1892-fabric8-planner.int.rdu2c.fabric8.io

**Pull request merged**